website/docs/concepts/event-bridge/rule.md

```diff
@@ -3,3 +3,102 @@ sidebar_position: 2
 ---
 
 # Rule
+
+[Event Bus Rules](https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-bus-to-bus.html) filter the event on a bus and then send those events to a target with optional transformation.
+
+Functionless allows typescript to be used when defining a rule.
+
+```ts
+const bus = new EventBus(stack, 'bus');
+const func = new Function<string, void>();
```

Comment:
this should have a `...` for args

Reply:
actually should we set a goal to have all code examples compile? maybe with `declare const` for code that distracts from the example